tests/test_sparkcompare.py

```diff
@@ -1665,84 +1761,84 @@ def test_rows_both_all_returns_all_rows_in_both_dataframes_for_differently_named
     expected_df = spark.createDataFrame(
         [
             Row(
+                accnt_purge=False,
                 acct=10000001234,
+                date_fld_base=datetime.date(2017, 1, 1),
+                date_fld_compare=datetime.date(2017, 1, 1),
+                date_fld_match=True,
```

Comment:
So essentially the order here was breaking the tests?

Reply:
Yeah it has to do the with the schema and how the union function works. I guess before it would ignore the order and now it has to. I’ll dig into this more but at least we know spark 3 works 😃